applications/MultilevelMonteCarloApplication/tests/momentEstimatorTests.py

```diff
@@ -0,0 +1,790 @@
+# Import python class test
```

Comment:
Are you using Kratos in this test? Shouldn't this test be inside xMC?

Reply:
We're checking an internal behavior, correct.

Reply:
Then, this test could be move inside xMC no? I would avoid adding to the Kratos testing something that does not contains Kratos.

Reply:
When we decided in April 2020 to integrate XMC within Kratos, i understood that we would have run xmc on Kratos CI, to see they are 100% compliant. Of course not all xmc tests can contain Kratos, so my understanding was to add also other tests.

Did you understand differently? Can you add something @RiccardoRossi and @roigcarlo ?

In case I misunderstood, I apologize.

Reply:
After talking with @RiccardoRossi and @roigcarlo , we agreed to keep current testing structure.